examples/PalmerPenguins.jl

```diff
@@ -0,0 +1,65 @@
+using PalmerPenguins, Plots
+const TABLE = PalmerPenguins.load()
+
+using DataFrames, Random
+df = dropmissing(DataFrame(TABLE))
+df[randperm(nrow(df)), :]
```

Comment:
This does not modify `df` although I guess you would like to permute the rows:
```suggestion
df = df[randperm(nrow(df)), :]
```